### ISSUE TITLE
No module named 'pyparsing'

### ISSUE THREAD

**Issue by deldesir** [chapmanjacobd/library#25]
**Describe the bug**
Upon running the `lb tubeadd` command with the provided YouTube video link (`https://www.youtube.com/watch?v=sqoOzGMqCQU`), the following error is encountered:

```
ModuleNotFoundError: No module named 'pyparsing'
```

This error suggests that the `pyparsing` module is missing or not installed in the environment where the `lb` command is executed.

**To Reproduce**
1. Execute the following command in the terminal:
   ```bash
   lb tubeadd https://www.youtube.com/watch?v=sqoOzGMqCQU
   ```
2. Observe the error message indicating the absence of the `pyparsing` module.

**Expected behavior**
The `lb tubeadd` command should execute successfully without encountering a `ModuleNotFoundError`. The necessary dependencies, including `pyparsing`, should be available in the environment.

**Desktop (please complete the following information):**
- OS: Ubuntu 24.04 (Noble Numbat)
- xklb version: 2.2.197

**Additional context**
Should be easily fixable running the following inside the xklb venv (but it failed):

```bash
pip install pyparsing
```

**Comment by chapmanjacobd** [comment]
Thanks for catching this! 

I released a new version (v2.2.198) that fixes this. I will keep this open until I add a test to prevent future mistakes like this from happening again.

**Comment by chapmanjacobd** [comment]
okay it looks like this should happen a lot less often now: https://github.com/chapmanjacobd/library/actions/runs/7335513984/job/19973528701
